validmind/tests/model_validation/ModelMetadataComparison.py

```diff
@@ -0,0 +1,37 @@
+# Copyright © 2023-2024 ValidMind Inc. All rights reserved.
+# See the LICENSE file in the root of this repository for details.
+# SPDX-License-Identifier: AGPL-3.0 AND ValidMind Commercial
+
+import pandas as pd
+from validmind.utils import get_model_info
+
+
+def ModelMetadataComparison(models):
+    """
+    Model Metadata Comparison
```

Comment:
Just a quick heads up that this one doesn't have a full description

Reply:
Good call out :)